### DISCUSSION TITLE
分享一个firefox安装方法(非临时安装，可账号同步)

### DISCUSSION THREAD

**Discussion by CoderCore** [ronggang/PT-Plugin-Plus#685] [Ideas]
前往 addons.mozilla.org 注册开发者账号，进入开发者中心，提交新的附加组件，选择我自己托管，待审核通过后，下载签名过的xpi文件即可正常安装，而且可以随账号同步。

**Comment by yjcn** [comment]
感谢老哥分享~  刚试了确实可以

**Comment by Raabo** [comment]
(下载签名过的xpi文件即可正常安装)这个步骤怎么弄的呢

**Comment by CoderCore** [comment]
> 
> 
> (下载签名过的xpi文件即可正常安装)这个步骤怎么弄的呢

找自己发布的插件列表，点进去查看所有版本

**Comment by Raabo** [comment]
> 
> 
> > (下载签名过的xpi文件即可正常安装)这个步骤怎么弄的呢
> 
> 找自己发布的插件列表，点进去查看所有版本

谢谢老哥

**Comment by Rhilip** [comment]
由于Wiki中已经补充该方法链接，故关闭

**Comment by lslqtz** [comment]
1. 降级至 Firefox ESR (可以使用 Firefox 命令行启动加上参数 --allow-downgrade 来降级 Firefox, 建议备份) 或升级至 Firefox Developer Edition;
2. 打开 about:config, 将 xpinstall.signatures.required 及 xpinstall.whitelist.required 设为 false;
3. 下载 GitHub Action 的最新 zip 构建, 解包, 并修改 manifest.json, 加入如下内容:
```
"browser_specific_settings": {
  "gecko": {
    "id": "your email"
  }
}
```
4. 将文件夹内的内容打包为 zip, 随后选择“从文件安装附加组件”;

参考链接: https://wiringbits.net/browser-extensions/2020/11/27/installing-unsigned-extensions-permanently-to-firefox.html
<img width="898" alt="截屏2024-01-24 03 53 57" src="https://github.com/pt-plugins/PT-Plugin-Plus/assets/18650291/9f1769bd-69c9-4193-aa60-90167ffa9fa1">
